linode/manager
# feat: [M3-6980] - Add DC Specific Pricing Invoice Support

## Description 📝
- Adds Invoice support for DC specific pricing 💵

## Major Changes 🔄
- Adds `region` column to Invoice Details, PDF Invoice, and CSV invoice

## Preview 📷

### Invoice Detail 🧾
![Screenshot 2023-08-28 at 4 39 42 PM](https://github.com/linode/manager/assets/115251059/d89fcc82-3c54-4fe2-8abf-c6a141243a91)

### CSV 🧾
![Screenshot 2023-08-28 at 4 40 10 PM](https://github.com/linode/manager/assets/115251059/24886f94-6f75-4cb2-9f6d-2e7d3aef64f9)

### PDF 🧾
![Screenshot 2023-08-28 at 4 40 41 PM](https://github.com/linode/manager/assets/115251059/163a7611-0d5f-4a6b-8df9-e138b3b3c691)


## How to test 🧪
- Turn the **MSW on** and the **DC Specific Feature Flag on**
- Visit `http://localhost:3000/account/billing/invoices/555`
  - Observe that there is a region column in the invoice (check PDF and CSV too)
- Verify you do not see the region column when the feature flag is **off**
